app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -812,6 +812,18 @@ export default Controller.extend({
       });
     },
 
+    cannotSeeGroupMention(mentions) {
```

Comment:
Just out of curiosity, why does this accept an array when we only ever call the function with a single item in the array?

Reply:
I do not know. I asked myself the same, but that is no longer the case with the refactor.